src/app/page/header/header.component.ts

```diff
@@ -7,6 +7,9 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HeaderComponent implements OnInit {
 
+  @Input() isMenuOpen = false;
+  @Output() menuToggled = new EventEmitter<boolean>();
```

Comment:
Bør nok omdøbes til bydeform, ligesom vi gør med `tap`, `click` etc.:
```suggestion
   @Output() menuToggle = new EventEmitter<boolean>();
```

Reply:
Og evt. til `ToggleMenu`, så passer pengene bedre med `onToggleMenu` a la `click` => `onClick`